modules/ort/src/main/java/org/eclipse/sw360/antenna/ort/workflow/analyzers/OrtResultAnalyzer.java

```diff
@@ -258,6 +260,14 @@ private String mapSourceUrl(JsonNode ortItem) {
         }
     }
 
+    private String mapHomepageUrl(JsonNode ortItem) {
```

Comment:
If it is maybe not present it is better to return an optional:
```suggestion
    private Optional<String> mapHomepageUrl(JsonNode ortItem) {
```

later one can consume that in the following way:
```java
        mapHomepageUrl(scanResult.get("results")).map(ArtifactSourceUrl::new).ifPresent(a::addFact)
```